backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/impl/HUPIItemProductDAO.java

```diff
@@ -487,6 +488,7 @@ public I_M_HU_PI_Item_Product retrieveMaterialItemProduct(
 				// Get specific Product first
 				.addColumn(I_M_HU_PI_Item_Product.COLUMNNAME_IsAllowAnyProduct, Direction.Descending, Nulls.Last) // Y first, N second
 				.addColumn(I_M_HU_PI_Item_Product.COLUMNNAME_M_Product_ID, Direction.Descending, Nulls.Last)
+				.addColumn(I_M_HU_PI_Item_Product.COLUMNNAME_IsDefaultForProduct, Direction.Descending, Nulls.Last)
```

Comment:
This was missing from Dragos's changes, but was necessary (at least for the batch add)